recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
Good point @tkelman. There is a package [ASL](https://packages.debian.org/sid/libs/libasl0) on both Debian and Fedora, but it's "Advanced Simulation Library," so there would be a name conflict if we packaged ASL there. As far as I can tell, ASL has not been packaged for either Debian or Fedora yet, since it is not a dependency of [Ipopt](https://packages.debian.org/sid/coinor-libipopt1v5) (which in fact means that the binary is not packaged, only the library). Hmm, perhaps we should instead change the name in Homebrew...

I'm cc'ing @dpo who did the packaging for Homebrew to make him aware of this discussion.